### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by MaxGdr**
This PR give some templates for :

- Issues
    - Feature Request
    - BugFix Request
    - Documentation
- Pull Request

In addition, I initialize a first version of a contributing.md as a guideline for future Shapash contributors

NOTE : We have to modify images URL's in the CONTRIBUTING.md before MERGE (They should be in MAIF/Shapash master branch)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+# Create your contribution to submit a pull request
+## Fork to code in your personal Shapash repo
+
+The first step is to get our MAIF repository on your personal GitHub repositories. To do so, use the "Fork" button.
+
+<img src="https://raw.githubusercontent.com/MaxGdr/shapash/contributing/docs/assets/images/contributing/shapash-fork.png" alt="fork this repository" />
+
+## Clone your forked repository
+
+<img align="right" width="300" src="https://raw.githubusercontent.com/MaxGdr/shapash/contributing/docs/assets/images/contributing/shapash-clone.png" alt="clone your forked repository" />
```
Comment (MaxGdr) [inline CONTRIBUTING.md:37]: Change URL with https://.../MAIF/shapash/...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+Before you open an issue, please check the current opened issues to insure there are no duplicate. Define if it's a feature or a bugfix.
+
+Next, the Shapash team, or the community, will give you a feedback on whether your issue must be implemented in Shapash, or if it can be resolved easily without a pull request.
+
+# Create your contribution to submit a pull request
+## Fork to code in your personal Shapash repo
+
+The first step is to get our MAIF repository on your personal GitHub repositories. To do so, use the "Fork" button.
+
+<img src="https://raw.githubusercontent.com/MaxGdr/shapash/contributing/docs/assets/images/contributing/shapash-fork.png" alt="fork this repository" />
```
Comment (MaxGdr) [inline CONTRIBUTING.md:33]: Change URL with https://.../MAIF/shapash/...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+Next step is now to create a Pull Request so the Shapash Team can add your changes to the official repository.
+
+## Create a Pull Request
+
+
+A pull request allows you to ask the Shapash team to review your changes, and merge your changes into the master branch of the official repository.
+
+To create one, on the top of your forked repository, you will find a button "Compare & pull request"
+
+<img src="https://raw.githubusercontent.com/MaxGdr/shapash/contributing/docs/assets/images/contributing/shapash-compare-pr.png" alt="pull request" />
```
Comment (MaxGdr) [inline CONTRIBUTING.md:105]: Change URL with https://.../MAIF/shapash/...

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+As you can see, you can select on the right side which branch of your forked repository you want to associate to the pull request. 
+
+On the left side, you will find the official Shapash repository.
+
+- Base repository: MAIF/shapash
+- Base branch: master
+- Head repository: your-github-username/shapash
+- Head branch: your-contribution-branch
+
+<img src="https://raw.githubusercontent.com/MaxGdr/shapash/contributing/docs/assets/images/contributing/shapash-pr-branch.png" alt="clone your forked repository" />
```
Comment (MaxGdr) [inline CONTRIBUTING.md:116]: Change URL with https://.../MAIF/shapash/...

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+Once you have selected the right branch, let's create the pull request with the green button "Create pull request".
+
+<img src="https://raw.githubusercontent.com/MaxGdr/shapash/contributing/docs/assets/images/contributing/shapash-pr-description.png" alt="clone your forked repository" />
```
Comment (MaxGdr) [inline CONTRIBUTING.md:120]: Change URL with https://.../MAIF/shapash/...

**Review by MaxGdr — COMMENTED** [review]
Todo: change image urls

**Review by TFA-MAIF — APPROVED** [review]

**Review by GAP01 — APPROVED** [review]

**Review by SebastienBidault — APPROVED** [review]

**Review by ThomasBouche — APPROVED** [review]

**Review by johannmartin — COMMENTED** [review]
Good Work Maxime !

There's just a link problem in CONTRIBUTING.md here when we open it :

[How to open an **issue** : **Bugfix** - **Feature request** - **Doc improvement**](#how-to-open-an-issue-:-Bugfix---Feature-request--Doc-improvement)

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+In the description, a template is initialized with all informations you have to give about what you are doing on what your PR is doing.
+
+Please follow this to write your PR content.
+
+
+## Finally submit your pull request
+
+Your pull request is now ready to be submitted. A member of the Shapash team will contact you and will review your code and contact you if need.
```
Comment (mougi81) [inline CONTRIBUTING.md:129]: need --> needed ?

**Review by mougi81 — APPROVED** [review]
Great !

**Review by yg79 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 6 above)
Comment (MaxGdr) [reply CONTRIBUTING.md:129]: 👍

**Pull request merged**